modules/bridge/src/main/ts/ephox/bridge/components/menu/FancyMenuItem.ts

```diff
@@ -4,12 +4,14 @@ import { Result, Fun } from '@ephox/katamari';
 export interface FancyMenuItemSpec {
   type: 'fancymenuitem';
   fancytype: string;
+  initData?: Record<string, unknown>;
```

Comment:
is `initData` optional because it is defaulted in the `FieldSchema`? If so, should `onAction` also be optional?

Reply:
There's a PR to address this: https://github.com/tinymce/tinymce/pull/6957